src/Tizen.NUI/src/internal/Size.cs

```diff
@@ -21,13 +21,11 @@
 
 namespace Tizen.NUI
 {
-    // Please do not use! This will be changed to internal class.
     /// <summary>
     /// A three-dimensional size.
     /// </summary>
-    /// <since_tizen> 3 </since_tizen>
+    /// <since_tizen> 5 </since_tizen>
```

Comment:
This class is provided from TizenFX version 5 so all <since_tizen> tags in this class should be updated.